### ISSUE TITLE
JAWs not reading tooltip content when tooltip is wrapped around icon/button

### ISSUE THREAD

**Issue by kejatura** [microsoft/fluentui#18520]
### Environment Information

- **Package version(s)**: v7/v8
- **Browser and OS versions**:  Edge Chromium

### Describe the issue:

After adding `aria-live= {"assertive}`, JAWs screen reader is not announcing tooltip content when TooltipHost wraps around child components (i.e Icon, or IconButton). But this works for NVDA and Narrator.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/kejatura/pen/bGqxOWp
(Provided nested Icon and Button examples, just comment/uncomment where appropriate)

#### Actual behavior:

JAWs (v.2021) not announcing the tool tip message.

#### Expected behavior:

JAWs should be able to announce the tool tip message.

### Documentation describing expected behavior

Code pen extracted from IconButton controls page on https://developer.microsoft.com/en-us/fluentui#/controls/web/button -> "Icon Button with Tooltip" section

**Comment by gouttierre** [comment]
@kejatura - Thanks for reaching out and filing this issue with the repro details. 

@ling1726 - Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley this works for NVDA and Narrator.

**Comment by smhigley** [comment]
It looks like the issue is that `aria-describedby` is on the TooltipHost, and not on the element that is being described (in this case, the button). That said, JAWS doesn't read descriptive text on default verbosity settings. You'd need to change that in the user settings to get the tooltip, even after the fix. The reason this works in Narrator and NVDA right now is because the button in that example also has a `title` attribute with the same text as the Fluent tooltip.

As an aside, please don't put `aria-live`, or any other live region attribute on a tooltip. That's a huge antipattern, and is almost certain to create more problems than it solves.
